StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/Vertical Main Screen/VerticalPaymentMethodListViewController.swift

```diff
@@ -145,9 +145,7 @@ class VerticalPaymentMethodListViewController: UIViewController {
                 paymentMethodType: paymentMethodType,
                 hasSavedCard: savedPaymentMethod?.type == .card, // TODO(RUN_MOBILESDK-3708)
                 promoText: incentive?.takeIfAppliesTo(paymentMethodType)?.displayText,
-                appearance: appearance,
-                // Enable press animation if tapping this transitions the screen to a form instead of becoming selected
-                shouldAnimateOnPress: delegate?.shouldSelectPaymentMethod(selection) == false
+                appearance: appearance
```

Comment:
Sorry I think I was unclear when we talked about this. Could we leave the current behavior as-is for vertical?  I think it was pretty intentional.